models/public/faceboxes/faceboxes.md

```diff
@@ -0,0 +1,85 @@
+# faceboxes
+
+## Use Case and High-Level Description
+
+FaceBoxes: A CPU Real-time Face Detector with High Accuracy. For details see the [repository](https://github.com/zisianw/FaceBoxes.PyTorch), [paper](https://arxiv.org/pdf/1708.05234.pdf)
+
+## Example
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Object detection                         |
+| Source framework                | PyTorch\*                              |
+
+## Performance
+
+## Input
+
+### Original model
+
+Image, name - `input.1` , shape - [1x3x1024x1024], format [BxCxHxW],
+   where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order - BGR.
+   Mean values - [104.0, 117.0, 123.0]
+
+### Converted model
+
+Image, name - `input.1` , shape - [1x3x1024x1024], format [BxCxHxW],
+   where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order - BGR.
+
+## Output
+
+### Original model
+
+Vectors of floating-point values - boxes and scores.
+ - boxes_out, name - 342, shape - [1x21824x4]
```

Comment:
It is difficult to call boxes, because you need to make some specific manipulations with them to get bbox coordinates.
we need more details:
 - bounding boxes deltas in format [BxAx4], where B - batch size, A - number of prior box anchors

Reply:
Fixed